example/Makefile

```diff
@@ -6,4 +6,4 @@ run: compile
 	@echo "Running WASM module..."
 	@cat fn-call.json \
 		| bky-as attest-fn-call \
```

Comment:
Just a heads up, the readme in this example specifies that this example relies on version `v0.1.0-beta.3`, which is no longer true with this update.

Reply:
Thank you for catching that.  I rolled it back (but also tested with the current version of main so that I could make sure that the log to host function works as expected).